recipe/meta.yaml

```diff
@@ -15,8 +15,10 @@ source:
     - ldflags.patch
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win and py27]
+  run_exports:
+    - {{ pin_subpackage('cfitsio', max_pin='x.x.x') }}
```

Comment:
AFAIK cfitsio versioning is MAJOR.MINOR, no .PATCH. Should this not be `'x.x'` then? (I have little experience with `pin_subpackage` so it's a genuine question).

Reply:
I don't know the gory details, but seeing as the CI passed, I infer that the pinning code is happy to have an over-specific pin.

More broadly, an `x.x.x` pin indicates that we have no confidence that the cfitsio libraries will maintain binary compatibility across (hypothetical) micro version updates, which I think is a fair assessment of the library's guarantees. If a 3.370.1 were ever released I would not be confident that it would be ABI-compatible with 3.370.

Reply:
That sounds reasonable. I will take this up with the upstream developer to double-check, but for now I'll merge this in.

Reply:
Like @pkgw said, it does not hurt to have more `.x` there, the problem is when we have less :smile: